src/shared/cryptsetup-util.h

```diff
@@ -41,7 +41,7 @@ DLSYM_PROTOTYPE(crypt_keyslot_destroy);
 DLSYM_PROTOTYPE(crypt_keyslot_max);
 DLSYM_PROTOTYPE(crypt_load);
 DLSYM_PROTOTYPE(crypt_resize);
-DLSYM_PROTOTYPE(crypt_resume_by_passphrase);
+DLSYM_PROTOTYPE(crypt_resume_by_volume_key);
```

Comment:
This was added in 2.3, which is above the baseline, so please add a check in meson.build at line 1247, and wrap this and the new bits in cryptsetup-util.c in an ifdef.

No need to use ifdefs in src/home/ though, simply make homed depend on `conf.get('HAVE_CRYPT_RESUME_BY_VOLUME_KEY') == 1` in meson.build at line 1563